tests/src/python/test_qgsserver_wms_getfeatureinfo.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 """QGIS Unit tests for QgsServer GetFeatureInfo WMS.
 
-From build dir, run: ctest -R PyQgsServerWMSGetFeatureInfo -V
+From build dir, run: ctest -R PyQgsServerWMSGetFeatureInfo\$ -V
```

Comment:
```suggestion
From build dir, run: ctest -R PyQgsServerWMSGetFeatureInfo\$ -V  # noqa: W605
```

Not very nice, but this is possible

Reply:
@Gustry Thanks for the info. Yeap, not looking good, wonder if it works though. Thank you!